LongWarOfTheChosen/Config/XComGameData_CharacterStats.ini

```diff
@@ -8867,62 +8867,174 @@ CharacterBaseStats[eStat_FlankingCritChance]=40
 CharacterBaseStats[eStat_ArmorMitigation]=5
 CharacterBaseStats[eStat_HP]=15
-+CharacterBaseStats[eStat_Offense]=180
+CharacterBaseStats[eStat_Offense]=80
+CharacterBaseStats[eStat_Will]=95
+
+[AdvPriestM1 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=0
+CharacterBaseStats[eStat_Dodge]=0
+CharacterBaseStats[eStat_HP]=6
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_PsiOffense]=20
+CharacterBaseStats[eStat_Will]=55
+
+
+[AdvPriestM1_Diff_0 X2CharacterTemplate]
+CharacterBaseStats[eStat_HP]=5
+CharacterBaseStats[eStat_Offense]=65
+CharacterBaseStats[eStat_PsiOffense]=15
+CharacterBaseStats[eStat_Will]=50
+
+[AdvPriestM1_Diff_2 X2CharacterTemplate]
+CharacterBaseStats[eStat_Dodge]=5
+CharacterBaseStats[eStat_HP]=8
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_PsiOffense]=20
+CharacterBaseStats[eStat_Will]=55
+
+[AdvPriestM1_Diff_3 X2CharacterTemplate]
+CharacterBaseStats[eStat_Dodge]=10
+CharacterBaseStats[eStat_HP]=8
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_PsiOffense]=25
+CharacterBaseStats[eStat_Will]=55
+
+
+[AdvPriestM2 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=0
+CharacterBaseStats[eStat_Dodge]=10
+CharacterBaseStats[eStat_HP]=12
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_PsiOffense]=25
+CharacterBaseStats[eStat_Will]=75
+
+
+[AdvPriestM2_Diff_0 X2CharacterTemplate]
+CharacterBaseStats[eStat_Offense]=70
+CharacterBaseStats[eStat_PsiOffense]=20
+CharacterBaseStats[eStat_Will]=70
+
+[AdvPriestM2_Diff_2 X2CharacterTemplate]
+CharacterBaseStats[eStat_Dodge]=15
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_PsiOffense]=25
+CharacterBaseStats[eStat_Will]=75
+
+[AdvPriestM2_Diff_3 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=5
+CharacterBaseStats[eStat_Dodge]=15
+CharacterBaseStats[eStat_Offense]=80
+CharacterBaseStats[eStat_PsiOffense]=30
+CharacterBaseStats[eStat_Will]=75
+
+
+[AdvPriestM3 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=5
+CharacterBaseStats[eStat_Dodge]=15
+CharacterBaseStats[eStat_HP]=16
+CharacterBaseStats[eStat_Offense]=80
+CharacterBaseStats[eStat_PsiOffense]=30
+CharacterBaseStats[eStat_Will]=95
+
+[AdvPriestM3_Diff_0 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=0
+CharacterBaseStats[eStat_Dodge]=10
+CharacterBaseStats[eStat_HP]=13
+CharacterBaseStats[eStat_Offense]=75
+CharacterBaseStats[eStat_PsiOffense]=25
+CharacterBaseStats[eStat_Will]=90
+
+[AdvPriestM3_Diff_2 X2CharacterTemplate]
+CharacterBaseStats[eStat_Defense]=5
+CharacterBaseStats[eStat_Dodge]=15
+CharacterBaseStats[eStat_HP]=18
+CharacterBaseStats[eStat_Offense]=80
+CharacterBaseStats[eStat_PsiOffense]=30
+CharacterBaseStats[eStat_Will]=95
+
+[AdvPriestM3_Diff_3 X2CharacterTemplate]
+CharacterBaseStats[eStat_HP]=20
+CharacterBaseStats[eStat_Dodge]=20
+CharacterBaseStats[eStat_Defense]=10
+CharacterBaseStats[eStat_Offense]=85
+CharacterBaseStats[eStat_PsiOffense]=35
+CharacterBaseStats[eStat_Will]=95
```

Comment:
Would you mind reordering this one? Or I can do it when I merge. Just need to move the HP one so it's in the same location relative to the entries for the other difficulties. Makes it hard to compare values at a glance right now.